catalog-rest-service/src/main/resources/ui/src/components/app-bar/Appbar.tsx

```diff
@@ -178,6 +191,12 @@ const Appbar: React.FC = (): JSX.Element => {
               />
             </div>
           </div>
+          {isModalOpen && (
```

Comment:
Use self-descriptive variable names. For instance, `isWhatsNewModalOpen` or `isFeatureModalOpen`